src/native/third_party/snappy.gyp

```diff
@@ -1,6 +1,15 @@
 # Copyright (C) 2016 NooBaa
 {
     'includes': ['common_third_party.gypi'],
+    'target_defaults': {
+        'conditions': [
+            ['node_arch=="s390x"', {
```

Comment:
As far as I can see also s390 should be included here although I don't know if used. 
CC @uweigand